Scripts/Main.py

```diff
@@ -0,0 +1,52 @@
+import Sorcery_Functions
+
+# Testing our SorceryFunctions module
+
+# The below if statement is added in case we want to input
+# arguments in the command line for testing the Sorcery function.
+# So it's mainly added for convenience, since an alternative
+# would be to manually edit this script. When command line
+# arguments are used, the Main function expects the first
+# argument to be a boolean, although if this is not the case
+# it will type cast the argument to type "bool"
+# if (__name__ == "__main__"):
+#     import sys
```

Comment:
I left this part of the code untouched for now since it is commented out, and, once you get done with testing the SorceryFunctions.py file, you can let me know if you just want me to remove that section. I guess the reason the import statement is in the if statement is because, if the arguments for the functions we're wanting to test aren't put straight into the command line, then the sys module isn't used, but that functionality isn't really needed anyway because I already streamlined the testing of functions in the SorceryFunctions.py file by creating a testSorceries() function in Main.py. I guess common practice is to import the modules that we need at the beginning of scripts, so I can see why this is an eyesore.